src/smart-app-banner/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface SmartAppBannerProps {
+    show: boolean;
+    link: string;
+    content: SmartAppBannerContentProps;
+    offset?: number | undefined;
+    className?: string | undefined;
+    onBannerDismiss: () => void;
+    onBannerPress?: () => void | undefined;
+}
+
+export interface SmartAppBannerContentProps {
+    title: string;
+    buttonLabel: string;
+    buttonAriaLabel: string;
```

Comment:
Are these the only 2 props required for the button?

Reply:
yes, button will have the same behaviour as onBannerPress